requirements.txt

```diff
@@ -2,5 +2,4 @@ ggplot2
 fmsb
 gridExtra
 dplyr
-reshape
-introdataviz
\ No newline at end of file
+reshape
```

Comment:
Do you need devtools for anything @haydeeartaza. I had really har time compiling devtools with the set of other requirement in that R version. I spent a lot of time and calibrated all the dependencise and versions so that the image builds. So if you really need devtools, I will have to start from scratch and then I would prefer to upgrade R (in a new issue).